gui/mod-manager.lua

```diff
@@ -1,365 +1,461 @@
+                end
+
+                dialogs.showInputPrompt("Enter new name", nil, nil, current.name, function(t)
+                    local existing_idx = find_preset_by_name(t)
+
+                    if existing_idx then
+                        if existing_idx == idx then
+                            return
+                        end
+
+                        dialogs.showYesNoPrompt(
+                            "Confirmation",
+                            "Overwrite " .. t .. "?",
+                            nil,
+                            function()
+                                remove_preset(existing_idx)
+                                rename_preset(idx, t)
+                                refresh_list()
+                            end
+                        )
+                    else
+                        rename_preset(idx, t)
+                        refresh_list()
+                    end
+                end)
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 0, b = 2, w = 15 },
+            key = "SELECT",
+            label = "Load",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                load_preset(idx)
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 16, b = 2, w = 15 },
+            key = "CUSTOM_D",
+            label = "Delete",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                dialogs.showYesNoPrompt(
+                    "Confirmation",
+                    "Delete " .. current.text .. "?",
+                    nil,
+                    function()
+                        remove_preset(idx)
+                        refresh_list()
+                    end
+                )
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 0, b = 0, w = 20 },
+            key = "CUSTOM_Q",
+            label = "Set as default",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                toggle_default(idx)
+                refresh_list()
+            end
+        },
+    }
 end
-function manager:onInput(keys)
 
-    if keys.LEAVESCREEN  then
-        self:dismiss()
-    else
-        self:inputToSubviews(keys)
+ModmanageScreen = defclass(ModmanageScreen, gui.ZScreen)
+ModmanageScreen.ATTRS {
+    focus_path = "mod-manager",
+    defocusable = false,
+}
+
+function ModmanageScreen:init()
+    self:addviews{
+        ModmanageMenu{}
+    }
+end
+
+ModmanageOverlay = defclass(ModmanageOverlay, overlay.OverlayWidget)
+ModmanageOverlay.ATTRS {
+    frame = { w=30, h=1 },
```

Comment:
```suggestion
    frame = { w=16, h=3 },
    frame_style = gui.MEDIUM_FRAME,
```

Reply:
Forgot this one: DFHack hotkeys need to be clearly labeled as coming from DFHack